panel/browser-panel.hpp

```diff
@@ -72,7 +72,8 @@ static inline QCef *obs_browser_init_panel(void)
 #ifdef _WIN32
 	void *lib = os_dlopen("obs-browser");
 #else
-	void *lib = os_dlopen("../obs-plugins/obs-browser");
+	// Need to un-hardcode this, as I couldn't get it to work otherwise
+	void *lib = os_dlopen("/usr/lib/obs-plugins/obs-browser.so");
```

Comment:
Would @tbodt's solution on macOS work here too?

https://github.com/obsproject/obs-browser/pull/197/files#diff-69dd8c025b947c895c80b4705d56a61d880c8841834614a0643b6cb727e4cd19R98-R99

Reply:
Didn't work for me.

Reply:
This library lookup is fixed now.